src/main/java/com/bravo/user/model/dto/ErrorDto.java

```diff
@@ -1,11 +1,12 @@
 package com.bravo.user.model.dto;
 
-import java.util.Set;
 import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Set;
 
 @Data
-@NoArgsConstructor
+@RequiredArgsConstructor
```

Comment:
This change makes sense.  If making this change, would adding the final or @NonNull decorations also be needed?

Reply:
good point, final modifier or @nonnull annotation is needed for required args.